src/resources/postcss/components/_top-bar.pcss

```diff
@@ -7,13 +7,14 @@
 .tribe-events {
 
 	.tribe-events-c-top-bar {
-		align-items: center;
-		display: flex;
-		flex-direction: row;
-		flex-wrap: wrap;
-		margin-bottom: var(--spacer-2);
+		display: none;
 
 		@media (--viewport-medium) {
+			align-items: center;
+			display: flex;
+			flex-direction: row;
+			flex-wrap: wrap;
+			margin-bottom: var(--spacer-2);
```

Comment:
`--spacer-2` or `--spacer-5`? also, be sure to pull the latest spacer changes and fix this here.

Reply:
I actually moved the styles to the desktop viewport, because now the top bar will be hidden with the new designs. So, I assume I should update that to `--spacer-3` as it was 2 before.

Reply:
oh, but that was for mobile. So, `--spacer-5` it is.